src/iop/temperature.c

```diff
@@ -258,22 +240,15 @@ void connect_key_accels(dt_iop_module_t *self)
   dt_accel_connect_slider_iop(self, "red", GTK_WIDGET(g->scale_r));
   dt_accel_connect_slider_iop(self, "green", GTK_WIDGET(g->scale_g));
   dt_accel_connect_slider_iop(self, "blue", GTK_WIDGET(g->scale_b));
-  dt_accel_connect_slider_iop(self, "green2", GTK_WIDGET(g->scale_g2));
+  dt_accel_connect_slider_iop(self, "emerald", GTK_WIDGET(g->scale_g2));
   dt_accel_connect_combobox_iop(self, "presets", GTK_WIDGET(g->presets));
```

Comment:
if we could change "presets" to "settings" to match combo label...

Reply:
Ugh. This made me look through the shortcuts dialog and notice that the automatically generated shortcuts for "preset"s sometimes generate spurious sub-sections when there is a / in the preset name. For example under channel mixer preset b w  artifacts boost.

If I ever get round to working on https://github.com/darktable-org/darktable/issues/5063 (all the clean-up I'm doing now is just ground work to not later on have to deal with all the exceptions) getting rid of the /-based accelerator organisation is high on the list of secondary objectives.